loadtests/src/test/scala/svp/ShieldedVulnerablePeople.scala

```diff
@@ -42,6 +42,13 @@ object ShieldedVulnerablePeople {
     .check(status.is(302)))
     .pause(Config.pauseMin, Config.pauseMax)
 
+  private val reqAddressLookupPost = exec(http("POST - Address lookup")
```

Comment:
this just looks like it has moved in the file? I don't see a change in the simulation file to accommodate the changes in the UI